draft-ietf-lamps-kyber-certificates.md

```diff
@@ -74,24 +83,26 @@ and will use placeholders until these are released.]
 
 # Introduction
 
-The US NIST PQC Project has selected the Kyber algorithm
-as winner of their PQC Project {{PQCProj}}. This
-algorithm is a Key Encapsulation Mechanism(KEM). NIST has also defined object identifiers
-for these algorithms (TODO insert reference).
-
-This document specifies the use of the Kyber algorithm
-in X.509 public key certifiates, see {{!RFC5280}}.
-It also specifies private key encoding.
-An ASN.1 module is included for reference purposes.
-
-These certificates could be used as Issuers in CMS where the public key
-is used to encapsulate a shared secret used to derive a symmetric key
-used to encrypt content in CMS
-\[EDNOTE: Add reference draft-perret-prat-lamps-cms-pq-kem\].
-To be used in TLS, these certificates could only be used as end-entity
-identity certificates and would require significant updates to the
-protocol
-\[EDNOTE: Add reference draft-celi-wiggers-tls-authkem\].
+Kyber is a key-encapasulation mechanism (KEM). This document specifies
+the use of the Kyber algorithm at three security levels, Kyber512,
+Kyber768, and Kyber1024, in X.509 public key certifiates; see
+{{!RFC5280}}. Public and private key encodings are alsospecified.
+
+## ASN.1 and Kyber Identifiers
+
+An ASN.1 module {{X680}} is included for reference purposes. Note that
+as per {{RFC5280}}, certificates use the Distinguished Encoding Rules;
+see {{X690}}. Also note that NIST defined the object identifiers for
+the Kyber algorithms in an ASN.1 modulle; see (TODO insert reference).
+
+## Applicability Statement
+
+Kyber certificates are used in protocols where the public key is used to
+encapsulate a shared secret used to derive a symmetric key used to
```

Comment:
Same as the other comment. Kyber doesn't allow you to pick the shared secret to encapsulate. Instead, perhaps: "[...] to generate and encapsulate [...]"

Reply:
nit: add a colon to start of list?
>This document specifies the use of the Kyber algorithm at three security levels: Kyber512, Kyber768, and Kyber1024,